modules/gem5/_posts/2000-04-01-assignment2.md

```diff
@@ -230,10 +276,15 @@ In your report answer the following questions.
 ## Submission
 
 As mentioned before, you are allowed to submit your assignments in **pairs** and in **PDF** format.
-You should submit your report on [gradescope](https://www.gradescope.com/courses/487868).
-In your report answer the questions presented in [Analysis and simulation](#analysis-and-simulation), [Analysis and simulation: Step I](#step-i), [Analysis and simulation: Step II](#step-ii), and [Analysis and simulation: Step III](#step-iii).
+You should submit your report on
+[gradescope](https://www.gradescope.com/courses/487868).
```

Comment:
This should be using the variables in `_data`

Reply:
done.